syntax = "proto3";

package echo;

service EchoService {
  rpc takeAllMetrics(Request) returns (Reply) {}
  rpc takeDefaultRangedValue(Request) returns (Reply) {}
  rpc callETLDataPipeline(Request) returns (Reply) {}
  rpc getViolationsForSLAManager(Request) returns (Reply) {}
  rpc getFutureViolationsForSLAManager(Request) returns (Reply) {}
}

// The request message containing the user's name.
message Request {
  string content = 1;
}

// The response message containing the greetings
message Reply {
  string content = 1;
}


//  repeated string list=1;   //Serve per fare le liste.